src/styles/App.scss

```diff
@@ -1,3 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;1,600&family=Roboto:wght@300;400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;1,600&family=Montserrat&display=swap');
+
 .app {
-  margin: 60px 100px 10px 100px;
+  margin: 10px 100px;
+  /* stylelint-disable */
```

Comment:
- [ ] please, for good CSS practices, kindly remove the disable stylint and fix it properly. This will help your work become clean.